modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/IgniteSqlFunctions.java

```diff
@@ -125,7 +125,12 @@ public static String toString(ByteString b) {
 
     /** LENGTH(VARBINARY|VARCHAR). */
     public static int length(Object b) {
-        return b instanceof ByteString ? octetLength((ByteString) b) : charLength((String) b);
+        return b instanceof ByteString ? SqlFunctions.octetLength((ByteString) b) : charLength((String) b);
+    }
+
+    /** OCTET_LENGTH(VARBINARY|VARCHAR). */
+    public static int octetLength(Object s) {
```

Comment:
it's better to have overloaded version for every supported type of parameters

Reply:
+1 for overload